src/main/resources/schemas/sdc-schemas/sdcml-contract.xsd

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would place "note to the teams" or other remarks ("haha") in a comment ("Author" is a bit misleading given that the XML was already there).